### PR TITLE
Save the Environment by reducing purge through retracting filament during every Filament change! Less Poop! No lower reliability! Very simple change!

### PR DESCRIPTION
**Pull request by LeonFisherSkipper**
### **TLDR: This change would save huge amounts of waste with no reduction in print quality or reliability. No drawbacks at all. Thousands of people have tested and confirmed that it works. For details look in printables link.**

**This makes the Bambu printers much more competitive against the Prusa XL in multi color printing.** 

**Please for mother Nature implement these changes to safe the environment from unnecessary waste!**

**Comparison Before (Left) and After (right) the changes** :

IMG-4320

**Details:**

**I posted a [printables profile](https://www.printables.com/model/582382-bambulab-profile-for-up-to-60-purge-reduction) which reduces the purge volumes** without negativ effects in september and since then thousands of people have tested it without issues. Many people on [reddit](https://www.reddit.com/r/BambuLab/comments/18c5sqh/comment/kc9gt03/?utm_source=share&utm_medium=web2x&context=3) recommend this. Also In a video by [Teaching Tech](https://youtu.be/p3ZIM3megIU?si=PhgUzG7-S1dT1NH6) the top comment recommends this change to safe filament. 

I have personally tested this with problematic filament combinations like PVA and Nylon and even than it works flawlessly on thousands of filament changes. 

DzzD also included this change in a recent update to his [Pull request](https://github.com/bambulab/BambuStudio/pull/1660), 
but I think his other changes require more testing to make sure that the reliability stays high and blops don't fall onto the bed etc.

**I believe my changes should be added in the mean time before the filament change is speed up like DzzD suggested, as my changes dont require testing and are incredibly easy to implement.** 

The only changes required are 8 lines of G-code in the Nozzle profiles and optionally that in the flushing-volume calculator a flat 50 mm3 is subtracted from every combination. So that instead of the default 280, the new default would be 230. The same for auto-calculated values, where if the autocalculated value was 467 for example until now, the new value would be 467-50=417.

A other user already opened a [pull request](https://github.com/bambulab/BambuStudio/pull/2516), but I wanted to post it myself, as I am the author of the original Printables Post and I can answer questions here and hope to get the changes implemented.

### CODE REVIEW

**Comment by iZonex** [comment]
@LeonFisherSkipper That pull request not for master branch? As I see 946 commits, that PR never will be accepted at all. Please only leave what must be changed with 1 commit.

**Comment by LeonFisherSkipper** [comment]
@iZonex I linked the wrong branch. I marked the location where the changes would go now.
